kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2761,6 +2761,14 @@ object KyuubiConf {
       .version("1.10.0")
       .fallbackConf(OPERATION_INCREMENTAL_COLLECT)
 
+  val ENGINE_SPARK_OPERATION_INCREMENTAL_COLLECT_SKIP_CANCEL_GROUP: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.spark.operation.incremental.collect.skip.cancel.group")
+      .doc("When true, the job group cancellation in executeStatement method " +
+        "will be skipped on incremental collect mode")
```

Comment:
rather than repeating information that is easily inferred from the configuration name, the documentation should be enriched to explain why should this configuration be introduced and when should it be adjusted.